Makefile

```diff
@@ -39,6 +41,7 @@ NEXTPNR   = $(DOCKERBIN) $(DOCKERARGS) hdlc/nextpnr:ecp5 nextpnr-ecp5
 ECPPACK   = $(DOCKERBIN) $(DOCKERARGS) hdlc/prjtrellis ecppack
 OPENOCD   = $(DOCKERBIN) $(DOCKERARGS) --device /dev/bus/usb hdlc/prog openocd
 VUNITRUN  = $(DOCKERBIN) $(DOCKERARGS) ghdl/vunit:llvm python3 ./run.py
+VERILATOR = $(DOCKERBIN) $(DOCKERARGS) verilator/verilator:latest
```

Comment:
Might want to use `hdlc/verilator` instead, which is 76.71 MB instead of 331.63 MB.
That's because the one from hdlc contains runtime dependencies only.
Should you find any issue with that container, we can update/enhance it.

The is also `hdlc/sim`, which contains both verilator and GHDL, should you need it.

By the way, `hdlc/` images are also available from `ghcr.io/hdl/` and `gcr.io/hdl-container/`.